oeps/best-practices/oep-0019-bp-developer-documentation.rst

```diff
@@ -150,6 +150,42 @@ ADRs
 .. _invaluable technique: https://www.thoughtworks.com/radar/techniques/lightweight-architecture-decision-records
 .. _edx/credentials/docs/decisions: https://github.com/edx/credentials/tree/master/docs/decisions
 
+A suggested ADR template::
+
+  ##################
+  0000 Title For ADR
+  ##################
+  .. The title should be a short noun phrases. For example, "0001 Django IDA" or "0009 LDAP for Multitenant Integration"; filename should be lowercase with punctuation removed and spaces replaced by dash
+
+  Status
+  ******
+
+  .. Standard statuses
+      - **Draft** if the decision is still preliminary and in experimental phase
+      - **Accepted** *(date)* once it is agreed upon
+      - **Superseded** *(date)* with a reference to its replacement if a later ADR changes or reverses the decision
+
+  Context
+  *******
+
+  .. This section describes the forces at play, including technological, political, social, and project local. These forces are probably in tension, and should be called out as such. The language in this section is value-neutral. It is simply describing facts.
+
+  Decision
+  ********
+
+  .. This section describes our response to these forces. It is stated in full sentences, with active voice. "We will …"
+
+  Options Considered
```

Comment:
I like the "background-decision-consequences" sections being tight together, and then adding more depth as needed.

Thus, I would move the "Options Considered" section below the "Consequences" section, and would add a comment at the end to encourage authors to add other sections as appropriate (like "Author Concerns", "Related Decisions", "Resources", etc.)